dbt/include/trino/macros/materializations/table.sql

```diff
@@ -1,58 +1,27 @@
-{% materialization table, adapter='trino' %}
+{% materialization table, adapter = 'trino' %}
   {%- set identifier = model['alias'] -%}
-  {%- set tmp_identifier = model['name'] + '__dbt_tmp' -%}
-  {%- set backup_identifier = model['name'] + '__dbt_backup' -%}
 
   {%- set old_relation = adapter.get_relation(database=database, schema=schema, identifier=identifier) -%}
   {%- set target_relation = api.Relation.create(identifier=identifier,
                                                 schema=schema,
                                                 database=database,
                                                 type='table') -%}
-  {%- set intermediate_relation = api.Relation.create(identifier=tmp_identifier,
```

Comment:
Can you please explain in the commit comment why it was considered that the intermediate relation was superfluos?